### DISCUSSION TITLE
How to ignore a dummy secret detection on a container scan?

### DISCUSSION THREAD

**Discussion by ChrisHolman** [aquasecurity/trivy#5904] [Q&A] [answered]
### Question

Trivy is exclusively utilised for its secret scanning features to scrutinise our containers.

At times, for testing purposes, fictitious keys have been introduced into these containers for testing purposes. 

We do not want to report on these fictitious keys. What method could I adopt to disregard these keys, preventing them from being detected by Trivy? 

I couldn't see in the documentation any reference to a ignore file, or similar. 

### Target

Container Image

### Scanner

Secret

### Output Format

JSON

### Mode

Standalone

### Operating System

ubuntu-latest

### Version

```bash
0.48.2
```

**Comment by itaysk** [comment, marked as answer]
disabling secret rules: https://aquasecurity.github.io/trivy/v0.48/docs/scanner/secret/#disable-rules
skipping files: https://aquasecurity.github.io/trivy/v0.48/docs/configuration/skipping/
